wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -641,23 +643,31 @@ public final class SessionManager: NSObject, SessionManagerType {
         return account
     }
 
-    private func selectInitialAccount(_ account: Account, launchOptions: LaunchOptions) {
+    private func selectInitialAccount(
+        _ account: Account,
+        launchOptions: LaunchOptions,
+        completion: @escaping (Bool) -> Void
+    ) {
         if let url = launchOptions[UIApplication.LaunchOptionsKey.url] as? URL {
             if (try? URLAction(url: url))?.causesLogout == true {
                 // Do not log in if the launch URL action causes a logout
-                return
+                return completion(false)
             }
         }
 
         guard !shouldPerformPostRebootLogout() else {
             performPostRebootLogout()
-            return
+            return completion(false)
         }
 
         loadSession(for: account) { [weak self] session in
-            guard let self, let session else { return }
+            guard let self = self, let session = session else {
```

Comment:
Please run the formatter, it will fix it automatically.

Reply:
Done!